### DISCUSSION TITLE
Mario Kart 8 mods for update 3.0.3

### DISCUSSION THREAD

**Discussion by TheManuel2** [StevensND/switch-port-mods#24] [Mod Request]
Hello.
I wonder if you might be able to update the MK8 mods from [here](https://github.com/theboy181/switch-ptchtxt-mods/tree/main/Mario%20Kart%208%20Deluxe/%5B0100152000022000%5D/3.0.1) to game version 3.0.3.
Best regards.

**Comment by StevensND** [comment]
No. **I will not update any mods made by this person.**

FYI: **This person is quite toxic and controversial when his mods are used** and only his selected ones can update his mods **without him or "his fans" telling you that you stole/leaked his mods**. (he claims his mods are private and puts them **behind a paywall** until some time passes and then makes them public).

However, [here's a guide ](https://github.com/StevensND/ghidra-port-mods-guide/blob/main/Ghidra/RyujinxSteps.md)to know how to update some mods. You can use it and update them by yourself.

**Reply by TheManuel2** [reply]
Thanks for the tip. I was unaware of this social dynamic. I’ll check out the guide.

**Reply by StevensND** [reply]
> Thanks for the tip. I was unaware of this social dynamic. I’ll check out the guide.

Ok. If you have any doubt you can contact me on Discord: stevenss. (point included)

**Reply by TheManuel2** [reply]
Thank you!

**Reply by StevensND** [reply]
> Thank you!

I saw [this post on Reddit](https://www.reddit.com/r/yuzu/comments/qfgqdi/comment/lqkgbts/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) btw

**Comment by TheManuel2** [comment]
Thank you!  Another Reddit user alerted me also. I’ll still be trying out your guide to attempt to update other mods.

**Comment by rakantoh** [comment]
the codes are the same that mk8 3.0.1, only change the nsobid

**Reply by StevensND** [reply]
> the codes are the same that mk8 3.0.1, only change the nsobid

[They're on GameBanana](https://gamebanana.com/mods/546686) anyway
